benchmarks/server_rendering_benchmark.rb

```diff
@@ -0,0 +1,63 @@
+# - `gem install duktape`
+# - Remove `JavaScriptCore` from RUNTIMES if you're not on mac
+# - `ruby -I lib  benchmarks/server_rendering_benchmark.rb`
+
+require 'react-rails'
+require 'duktape'
+require 'benchmark'
+
+SLOW_COMPONENT = "
+var SlowComponent = React.createClass({
+  render: function() {
+    var rand = 0
+    for (var i = 0; i < 10000; i++) {
+      rand = rand + (Math.random() * i)
+    }
+    return React.createElement('h1', rand + ' :)')
+  }
+})
+"
+
+REACT_JS_PATH = File.expand_path("../../vendor/react/react.js", __FILE__)
+JS_CODE = File.read(REACT_JS_PATH) + SLOW_COMPONENT
+
+React::ServerRendering.renderer = React::ServerRendering::ExecJSRenderer
+React::ServerRendering.renderer_options = {code: JS_CODE}
+React::ServerRendering.pool_timeout = 1000
+
+def test_runtime(runtime, renders:, pool_size:, threaded:)
```

Comment:
I meant with the default values to be explicit- nil. Just easier to read, nothing specific :-)